src/MakePotCommand.php

```diff
@@ -53,18 +60,20 @@ class MakePotCommand extends WP_CLI_Command {
 	 * : Name of the resulting POT file.
 	 *
 	 * [--slug=<slug>]
-	 * : Plugin slug. Defaults to the source directory's basename.
+	 * : Plugin or theme slug. Defaults to the source directory's basename.
 	 *
 	 * [--domain=<domain>]
 	 * : Text domain to look for in the source code. Defaults to the plugin/theme slug.
 	 *
-	 * [--exclude=<paths>]
-	 * : Include additional ignored paths as CSV (e.g. 'tests,bin,.github').
-	 *
+	 * [--merge[=<file>]]
+	 * : Existing POT file file whose content should be merged with the extracted strings.
 	 * By default, the following files and folders are ignored: node_modules, .git, .svn, .CVS, .hg, vendor.
```

Comment:
Merging seems to have messed up the docblocks. `--merge` & `--exclude` have the wrong explanations.

Reply:
This will also need a refresh of the README.md once fixed.

Reply:
Thanks, fixed now!